mail_autosubscribe/models/models.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Camptocamp (http://www.camptocamp.com).
+# @author Iván Todorovich <ivan.todorovich@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class BaseModel(models.AbstractModel):
+    _inherit = "base"
+
+    @api.model
+    def _message_get_autosubscribe_followers(self, partners):
+        return (
+            self.env["res.partner"]
+            .sudo()
+            .search(
+                [
```

Comment:
move this domain out to an helper method so that it can be easily overridden by extending modules like on #621 (to be renamed as `mail_autosubscribe_partner_category`)

Reply:
done :+1: